weave-js/src/core/ops/domain/artifactTag.ts

```diff
@@ -0,0 +1,13 @@
+import * as OpKinds from '../opKinds';
+
+const artifactTagArgTypes = {
+  artifactTag: 'artifactTag' as const,
+};
+
+export const opArtifactTagName = OpKinds.makeStandardOp({
+  hidden: true,
+  name: 'artifactTag-name',
+  argTypes: artifactTagArgTypes,
+  returnType: inputTypes => 'string',
+  resolver: ({artifactTag}) => artifactTag.name,
+});
```

Comment:
note: largely adapted from operators for artifact *aliases* from: https://github.com/wandb/weave/blob/69710f9342db3b0fbb3f46ad64b3d380296ef71c/weave-js/src/core/ops/domain/artifactAlias.ts#L3-L13